ckb-contracts/lib/eth-bridge-lockscript-lib/src/lib.rs

```diff
@@ -0,0 +1,75 @@
+#![cfg_attr(not(feature = "std"), no_std)]
+
+mod actions;
+pub mod adapter;
+pub mod debug;
+
+pub use adapter::Adapter;
+
+use adapter::BridgeCellDataTuple;
+
+cfg_if::cfg_if! {
+    if #[cfg(feature = "std")] {
+    } else {
+        extern crate alloc;
+    }
+}
+
+#[cfg(target_arch = "riscv64")]
+pub fn verify() -> i8 {
+    let chain = adapter::chain::ChainAdapter {};
+    _verify(chain)
+}
+
+pub fn _verify<T: Adapter>(data_loader: T) -> i8 {
+    let cell_data_tuple = data_loader
+        .load_input_output_data()
+        .expect("inputs or outputs cell num invalid");
+
+    debug!("cell_data_tuple {:?}", cell_data_tuple);
+
+    match cell_data_tuple {
+        BridgeCellDataTuple(Some(input_data), Some(output_data)) => {
+            actions::verify_mint_token(data_loader, input_data.as_slice(), output_data.as_slice())
+        }
+        _ => panic!("input and output should not be none"),
+    }
+}
```

Comment:
toCKB use `DataTuple` to figure out which tx (among 17 txs) the script shoud verify.

But here, bridge-lockscript only be executed in mint-tx, so current design may be inappropriate